utils/cargo-gbuild/src/bin/cargo-gbuild.rs

```diff
@@ -0,0 +1,51 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2024 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use anyhow::Result;
+use cargo_gbuild::GBuild;
+use ccli::{clap, App, Parser};
+
+/// Command `gbuild` as cargo extension.
+#[derive(Parser)]
+#[clap(author, version)]
+#[command(name = "cargo_gbuild")]
```

Comment:
```suggestion
#[command(name = "cargo-gbuild")]
```
Usually cargo subcommands are in this format

Reply:
https://github.com/gear-tech/gear/commit/5ac39db2f0d77c8509cad50e0ce03c842abb81d8 converting it to library is for auto logging detection, now got it back to `cargo-gbuild`